package.json

```diff
@@ -17,6 +17,7 @@
     "react-leaflet": "^3.2.5",
     "react-scripts": "4.0.3",
     "react-search-autocomplete": "^5.4.4",
+    "weather-icons": "^1.3.2",
```

Comment:
Do we need this dependency?
cuz we already have `@mui/icons-material` and we can leverage this to use the cloud icons maybe?